core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/ClusterBootstrapper.java

```diff
@@ -349,6 +363,37 @@ public void bootstrapNewNodes(Long stackId, Set<String> upscaleCandidateAddresse
         }
     }
 
+    public void rotateSaltPassword(Stack stack) {
+        if (!stack.isAvailable()) {
+            throw new BadRequestException("Rotating salt password is only available for stacks in available status");
+        }
+        if (!isChangeSaltuserPasswordSupported(stack)) {
+            throw new BadRequestException("Rotating salt password is not supported with your image version, " +
+                    "please upgrade to an image with salt-bootstrap version >= 0.13.6");
```

Comment:
If this exception ends up on the UI of the CP, it might worth mentioning that the version info is available in the image catalog. If it is caught internally, feel free ignore this comment.

Reply:
I thought the salt-bootstrap version is available on the image catalog UI, but no, only in the catalog. So it may be worth rephrasing both this and the freeipa message.